docs/UPGRADING_1.9.0.md

```
@@ -0,0 +1,75 @@
+# Upgrading to 1.9.0
+
+Make sure you have a current backup of your database and files before upgrading.
+
+## Table of Contents
+
+- [Summary of Changes](#summary-of-changes)
+- [Security Fixes](#security-fixes)
+  - [MCP Server Dependency Update](#mcp-server-dependency-update)
+  - [CVE Coverage](#cve-coverage)
+- [BinkP](#binkp)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)